### PR TITLE
Release 7.1.0

### PR DESCRIPTION
**Pull request by dhoppe**
Fixes: #562

### CODE REVIEW

**Comment by rwaffen** [comment]
bump? How is it going with the release?

**Comment by rwaffen** [comment]
bump v2

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v7.0.1](https://github.com/voxpupuli/puppet-php/tree/v7.0.1) (2020-01-21)
```
Comment (bastelfreak) [inline CHANGELOG.md:7]: @dhoppe ah we should probably regenerate this?

Comment (dhoppe) [reply CHANGELOG.md:7]: I will take care of it.

**Comment by rwaffen** [comment]
many thanks ❤️

**Pull request merged**